eng/Directory.Packages.props

```diff
@@ -195,6 +195,7 @@
     <PackageVersion Include="System.IO.Pipelines" Version="$(SystemIOPipelinesVersion)" />
     <PackageVersion Include="System.IO.Pipes.AccessControl" Version="5.0.0" />
     <PackageVersion Include="System.Memory" Version="4.5.5" />
+    <PackageVersion Include="System.Numerics.Vectors" Version="4.5.0" />
```

Comment:
Will this work in VS insertions? Won't it cause builds to fail for some reason? Should we do a validation run?

Reply:
@jjonescz this vsix isn't actually used for insertion - it uses one of Microsoft.CodeAnalysis.Compilers.x64/.arm64/.x64.vsix

However weirder things have broken the VS build so I can run a val insertion.

Reply:
ah, thanks for explanation; I think it's fine then, we can always revert :)